app/src/main/java/camp/nextstep/edu/github/di/AppComponent.kt

```diff
@@ -0,0 +1,22 @@
+package camp.nextstep.edu.github.di
+
+import android.content.Context
+import dagger.BindsInstance
+import dagger.Component
+import javax.inject.Singleton
+
+/**
+ * android-github
+ * @author jaesung
+ * @created 2023/04/24
+ */
+@Component(modules = [AppSubcomponents::class])
```

Comment:
```suggestion
@Component(modules = [ActivityModule::class])
```
SubComponent와 관계성을 맺기 위해 만든 클래스로 보이는데요! modules에 대입하는 Dagger 구조에 따른 네이밍으로 `XXXModule`로 변경하는건 어떻게 생각하시나요??